package.json

```diff
@@ -14,6 +14,7 @@
     "static-analysis": "run-p check-types check-format check-lint"
   },
   "dependencies": {
+    "@babel/runtime": "^7.13.10",
```

Comment:
Why we need to install `@babel/runtime` ?

Reply:
removed